src/DDTrace/Integrations/Laravel/V4/LaravelProvider.php

```diff
@@ -93,6 +94,7 @@ public function boot()
         }
 
         PDOIntegration::load();
+        ElasticSearchIntegration::load();
```

Comment:
As stated above, we might want to wrap these with `class_exists('Elasticsearch\Client')` checks. :)

Reply:
Also - it just occurred to me that someone might be using the Laravel integration but a different version of `Elasticsearch\Client` than v1. Not sure how we want to check that off the top of my head. :)

Reply:
This is also something that we need to discuss. Ideally, in the spirit of simplifying onboarding, we should detect ourself the version. In this case I added some guards to protect us from crashes: e.g.
```        $class = 'Elasticsearch\Namespaces\\' . $namespace;
        if (!method_exists($class, $name)) {
            return;
        }```

In my mind each integration should have a `DDTrace\Integrations\ElasticSearchIntegration::load()` which either guess from `xxxx_exist` or read from configuration and delegates the actuall loading logic to `DDTrace\Integrations\Vx\ElasticSearchIntegration::load()`. But this comes just from the top of my head.